src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package task;
+
+public class Deadline extends Task {
+    protected String dueTime;
+    private static final int TASK_NAME_INDEX = 0;
+    private static final int DUE_DATE_INDEX = 1;
+
+    public Deadline(String[] deadlineInfo) {
+        this(deadlineInfo[TASK_NAME_INDEX],
+                deadlineInfo[DUE_DATE_INDEX]);
+    }
+    Deadline(String description, String dueTime) {
+        super(description);
+        this.dueTime = dueTime;
+    }
+
+    @Override
+    public String toString() {
+        String status = isDone ? "X" : " ";
```

Comment:
Avoid usage of magic strings as much as possible

Reply:
Thank you, will refactor the magic strings!